x-pack/plugin/build.gradle

```diff
@@ -47,25 +47,30 @@ tasks.named("testJar").configure {
 }
 
 // location for keys and certificates
-File keystoreDir = file("$buildDir/keystore")
-File nodeKey = file("$keystoreDir/testnode.pem")
-File nodeCert = file("$keystoreDir/testnode.crt")
+File extraResourceDir = file("$buildDir/keystore")
```

Comment:
I don't get this.  
Why is the directory named `keystore` if it contains general resources?

Reply:
Good catch! It's a leftover piece. Update to `extra_resource`.